### PR TITLE
README: Explicit the main requirements

### PR DESCRIPTION
**Pull request by ErwanAliasr1**
When people discover the project, it's cool to give a quick overview of what are the dependencies to make this project being used.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,3 +55,18 @@ If a single output file is provided, **hwgraph** plots for each benchmark :
 If multiple output files are passed as arguments, and only if they were generated with the same script file, **hwgraph** will compare for each benchmark the performance metrics.
 
 For more details, see the specific documentation.
+
+# Requirements
+## Mandatory
+- python >= 3.9
```
Comment (anisse) [inline README.md:61]: Don't we want to document the runtime python dependencies? Or maybe link to requirements/base.in ?

**Comment by ErwanAliasr1** [comment]
I asked myself the same question.
Maybe linking is better

Le jeu. 23 mai 2024 à 16:44, Anisse Astier ***@***.***> a
écrit :

File: README.md (hunk 1 above)
Comment (ErwanAliasr1) [reply README.md:61]: I asked myself the same question.
Maybe linking is better

Comment (anisse) [reply README.md:61]: Let's add the link in another commit/PR.

**Review by anisse — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ErwanAliasr1) [reply README.md:61]: Why in another PR ?

**Review by beorn- — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (anisse) [reply README.md:61]: Just in case. Works with this PR.

**Review by anisse — APPROVED** [review]

**Pull request merged**